### ISSUE TITLE
Featured Projects View

### ISSUE THREAD

**Issue by rebecca-allegar** [Project-Heimdall/home#195]
Have a Featured Projects section on the main Stage Page.  These should be our "Gold Standard" projects that we have self-identified.

**Comment by herrjosua** [comment]
Isn't this the `Popular` and `Healthiest Projects`?

**Comment by rebecca-allegar** [comment]
This is a new category of projects that we identify on our own .  The gold-standard projects that we can choose which ones to showcase, not necessarily the healthiest or the most popular.

**Comment by herrjosua** [comment]
So how are we defining this `gold-standard` projects? Based on my current understanding these would be the healthiest projects.

**Comment by herrjosua** [comment]
Updated design. `HTML` and `LESS` styles have been modified on branch 195.
![screen shot 2017-01-12 at 11 26 02](https://cloud.githubusercontent.com/assets/1341786/21902475/07369efc-d8ca-11e6-9648-61b80d9a8e58.png)

**Comment by herrjosua** [comment]
`HTML` and `LESS` styles have been updated. Currently pulling data from the `Popular` tab. Need to discuss what makes a project featured, and how is a project made to be featured. config file?

**Comment by skim012** [comment]
Featured Projects should include:  1) Attune baseline and FirsNet branch on EGH  contact: Ben R. for more info  2) EIC MVP contact: Andre S.  3) ODP contact: Tony C.  4) Open Data contact: Jin T.  ... more to follow...

**Comment by onetrickwolf** [comment]
Working on this no longer blocked.

**Comment by onetrickwolf** [comment]
I added a field to the stage config (stageConf.js) that is an array of IDs for the featured projects. This file is not compiled and can be edited directly if needed (but a PR should also be made with the changes so it doesn't get overwritten on build).

I added a couple of heimdall projects and projects @rebecca-allegar mentioned to test it. Ideally you'd want 6 featured projects to make the layout even but it can take any number.
